modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -0,0 +1,81 @@
+[[contact-methods-salt]]
+= Salt Bundle
+
+== What is Salt Bundle?
+
+The Salt Bundle is a single binary package containing Salt Minion, Python 3, required Python modules and libraries.
+
+As the Salt Bundle is shipped with Python 3 and all the requirements for Salt to run, the Salt Bundle is not using the Python version coming from the system, so it can be installed on clients which doesn't meet the requirements for a given Salt version.
+
+It is also possible to install Salt Bundle on the system where Salt Minion is being used already to manage that client by the different Salt Master than the {productname} Server.
+
+== Client Registration with Salt Bundle as a Minion
+
+The registration method with the Salt bundle is the recommended registration method as long as it covers your use case.
+This section explains the advantages and limitations of the current implementation.
+The Salt bundle is provided as the [package]``venv-salt-minion`` that consists of Salt, Python, and the Python modules Salt depends on.
+Bootstrapping with {webui} is using Salt bundle as well, so bootstrapping with {webui} is not Python dependant.
+Using the Salt bundle, it is no longer needed that the client provides the required Python version or any Python modules.
+
+If you bootstrap new clients, registration with the Salt bundle is the default method.
+You can switch existing clients to the Salt bundle method.
+If you switch, the [package]``salt-minion`` package and its Python version and the Python modules will stay installed.
+
+=== Switching from Salt Minion to Salt Bundle
+
+The Salt state [literal]``util.mgr_switch_to_venv_minion`` is available to switch from [package]``salt-minion`` to [package]``venv-salt-minion``.
+It is recommended to switch to [package]``venv-salt-minion`` in two steps to avoid trouble with shifting processes:
```

Comment:
This part was there since the initial PR with Salt Bundle documentation: https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/registration-methods-saltbundle.html
Just in the other section. I haven't touched this part.

Reply:
Slightly rephased it and added extra section with the warning that the state apply could fail in case of running the second step with skipping the first.